### ISSUE TITLE
Warning TraktApi was never awaited

### ISSUE THREAD

**Issue by gateriderz** [dylandoamaral/trakt-integration#92]
Since the upgrade to Home Assistant 2024.1, we are now getting the following warning with Trakt-Integration 0.8.6:

```
2024-01-10 09:05:28.180 WARNING (MainThread) [py.warnings] /usr/src/homeassistant/homeassistant/helpers/update_coordinator.py:256: RuntimeWarning: coroutine 'TraktApi.fetch_upcomings' was never awaited
  return await self.update_method()

2024-01-10 09:05:28.181 WARNING (MainThread) [py.warnings] /usr/src/homeassistant/homeassistant/helpers/update_coordinator.py:256: RuntimeWarning: coroutine 'TraktApi.fetch_recommendations' was never awaited
  return await self.update_method()

2024-01-10 09:05:28.181 WARNING (MainThread) [py.warnings] /usr/src/homeassistant/homeassistant/helpers/update_coordinator.py:256: RuntimeWarning: coroutine 'TraktApi.fetch_next_to_watch' was never awaited
  return await self.update_method()

```

Let me know if I can help testing.

Thanks

**Comment by dylandoamaral** [comment]
Fixed v0.8.7

**Comment by gateriderz** [comment]
Thank you.  Works!
